src/sqlite3gen.cpp

```diff
@@ -600,8 +1027,8 @@ static void insertMemberFunctionParams(int id_memberdef, const MemberDef *md, co
 
       if (!a->attrib.isEmpty())
       {
-        bindTextParameter(params_select,":attributes",a->attrib.data());
-        bindTextParameter(params_insert,":attributes",a->attrib.data());
```

Comment:
Early on I superstitously added many .data() calls while trying to fix junk data in columns. Once I had that working, I cut out the ones that seemed unnecessary (including some that were already present).